docs/modules/operation/pages/search/search.adoc

```diff
@@ -0,0 +1,95 @@
+[[search-functionality]]
+= Search
+
+{page-component-title} provides two ways to search in the web UI: through the search bar and the *Search* view.
+
+== Search bar
+
+The search bar uses keywords to return both nodes that match the keyword(s) and a list of actions you can perform that relate to the keyword.
+Press the [shift] key twice or click in the search bar to activate it for typing.
+
+image::search/search-sample.png[]
+
+Note that the search results for nodes also display any xref:meta-data.adoc#metadata-overview[metadata] associated with the node, such as a label, location, admin user, and so on.
```

Comment:
That's not correct. The search results do only contain metadata information for the matches found in metadata. For instance if you search for 'foo' and you find a node due to a match in its name 'foobar' no metadata is displayed at all. But if you search for 'foo' and a match is found for a node 'bar' in its metadata customkey='somefoo' it is displayed. So, the idea for the feature "Customizing search results" below, was to give an user an option to define at least on line of information, that will be displayed for each node regardless how the match was found.

Reply:
Thank you, @christianpape for clarifying that. I will update.